extensions/mssql/src/objectExplorerNodeProvider/objectExplorerNodeProvider.ts

```diff
@@ -299,31 +289,34 @@ class RootNode extends TreeNode {
 }
 
 class DataServicesNode extends TreeNode {
-	private children: TreeNode[];
-	constructor(private _session: Session, private context: TreeDataContext, private nodePath: string) {
```

Comment:
Please revert so we defined things as private in constructor where possible. That's consistently used across our codebase and reduces overall lines of code to read. Thanks!

Reply:
fixed.